src/webgpu/api/validation/resource_usages/textureUsageInPassEncoder.spec.ts

```diff
@@ -431,24 +475,28 @@ g.test('subresources_and_binding_types_combination_for_color')
       });
       pass.endPass();
     } else {
-      const pass = t.beginSimpleRenderPass(
-        encoder,
-        type1 === 'render-target' ? view1 : t.createTexture().createView()
-      );
+      const pass = compute
+        ? encoder.beginComputePass()
+        : t.beginSimpleRenderPass(
+            encoder,
+            type1 === 'render-target' ? view1 : t.createTexture().createView()
+          );
 
       // Create bind groups. Set bind groups in pass directly or set bind groups in bundle.
       const storageTextureFormat0 = type0 === 'sampled-texture' ? undefined : 'rgba8unorm';
       const bindGroup0 = t.createBindGroup(0, view0, type0, dimension0, storageTextureFormat0);
       if (binding0InBundle) {
-        t.createAndExecuteBundle(0, bindGroup0, pass);
+        assert(pass instanceof GPURenderPassEncoder);
+        t.createAndExecuteBundle(0, bindGroup0, pass as GPURenderPassEncoder);
```

Comment:
nit: `as` shouldn't be needed because `assert` already constrains the type.

```suggestion
        t.createAndExecuteBundle(0, bindGroup0, pass);
```